STM32/DC/Core/Src/DCBoard.c

```diff
@@ -336,6 +341,35 @@ static volatile uint32_t* getTimerCCR(int pinNumber)
     }
 }
 
+/*!
```

Comment:
In line 331 the comment should be updated to V3.2 and of course test that that is in fact true for a V3.2 :=)